scripts/devel/build_and_install.py

```diff
@@ -77,7 +117,22 @@ def main(argv=sys.argv[1:]):
         if args.clean_after:
             clean_workspace(args.workspace_root)
 
-    return rc
+    if not args.run_abichecker:
+        return rc
+
+    with Scope('SUBSECTION', 'use abi checker'):
+        rc = call_abi_checker(
+            [args.workspace_root],
+            args.ros_version,
+            args.rosdistro_name,
+            env)
+    # Never fail a build because of abi errors but make them
+    # unstable by printing MAKE_BUILD_UNSTABLE. Jenkins will
+    # use a`plugin to make it
+    if rc != 0:
+        print('MAKE_BUILD_UNSTABLE')
```

Comment:
Since it appears that we control the parser for this string. Can we use a more precise one like "ABI_CHECK_FAILED" so that it's immediately clear where to look for the failure?

Reply:
I can easily move the print to something more verbose. The problem is that we will lose the option of reusing the same snippet of code and have to create one every time we need to use it. The message happens just after we display the error: http://18.206.154.49/job/Ddev__gazebo_ros_pkgs__ubuntu_bionic_amd64/13/parsed_console/, I think it's pretty easy to find where it comes from. Anyway I'm not strongly against changing it.

Reply:
I went back and looked at a few ABI failing jobs and this string is not as forward in the UI as I expected it to be so I'm less concerned.